CorsixTH/Lua/epidemic.lua

```diff
@@ -261,16 +262,23 @@ function Epidemic:checkNoInfectedPlayerHasLeft()
     if infected_patient.going_home and not infected_patient.cured and
         px and py and not self.hospital:isInHospital(px,py) then
       -- Patient escaped from the hospital, discovery is inevitable.
-      self.result_determined = true
-      if not self.inspector then
-        self:spawnInspector()
-      end
       self:finishCoverUp()
       return
     end
   end
 end
 
+--[[ Check for conditions that no any uncured infected patients
+left in hospital. If so then epidemic must end earlier than the
+length of the timer.]]
+function Epidemic:checkNoInfectedPatients()
```

Comment:
I'm fine with this, trying to add clarity to this function name any more likely will just make things worse.